docs/tutorials/t1.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Running T<sub>1</sub> Experiments with Qiskit"
+    "# T<sub>1</sub>"
```

Comment:
Inconsistent styling of `$T_1$` versus just `T1` in this tutorial needs to be updated